packages/portfolio-contract/tools/portfolio-actors.ts

```diff
@@ -30,10 +32,11 @@ import {
   type StatusFor,
 } from '../src/type-guards.js';
 import type { WalletTool } from './wallet-offer-tools.js';
-import { NonNullish } from '@agoric/internal';
 
 const { fromEntries } = Object;
-const range = (n: number) => [...Array(n).keys()];
+
+assert.equal(ROOT_STORAGE_PATH, 'orchtest');
+const stripRoot = (path: string) => path.replace(/^orchtest\./, '');
```

Comment:
If the input path lacks an "orchtest." prefix, would that be expected (and thus appropriate to pass through) or a bug? If the latter, it seems like another use case for #9996:
```suggestion
assert.equal(ROOT_STORAGE_PATH, 'orchtest');
const ROOT_STORAGE_PATH_PREFIX = `${ROOT_STORAGE_PATH}.`;
const stripRoot = (path: string) => {
  assert(path.startsWith(ROOT_STORAGE_PATH_PREFIX), path);
  return path.slice(ROOT_STORAGE_PATH_PREFIX.length);
}
```

But it's testing code anyway, so this is just an observation.

Reply:
It started as test code, but `/tools/` can be used by clients too, so this is something to think about.

Reply:
> If the input path lacks an "orchtest." prefix, would that be expected (and thus appropriate to pass through)

yes; the normal case is `published.ymax0...`